core/arch/arm/kernel/link.mk

```diff
@@ -131,12 +131,20 @@ endef
 $(link-out-dir)/version.o:
 	$(call gen-version-o)
 
+-include $(link-out-dir)/.tee.elf.cmd
+define check-link-objs
+$(if $(strip $(filter-out $(link-objs), $(old-link-objs))
+	     $(filter-out $(old-link-objs), $(link-objs))), FORCE_LINK := FORCE)
+endef
+$(eval $(call check-link-objs))
```

Comment:
I fear this will not force relink if one updates an already existing TA ELF file.
Maybe it could be more simple to simple force relink if EARLY_TA_PATH is defined.